lib/references.ts

```diff
@@ -56,13 +64,49 @@ export const uri = {
     nav_services: uriExtender(`${faaHq}/ato/service_units/techops/navservices`),
   }),
 
-  far: (part: number, section?: number) => {
-    if (typeof section === 'undefined') return cornell14CFR + `/part-${part}`;
-    return cornell14CFR + `/${part}.${section}`;
-  },
-
-  farAppendix: (part: number, letter: string) => {
-    return cornell14CFR + `/appendix-${letter}_to_part_${part}`;
```

Comment:
This has been deleted, its functionality rolled into the `far` method via the new `options` argument